R/morse.R

```diff
@@ -341,28 +363,26 @@ NULL
 NULL
 
 #' Survival datasets for \emph{Gammarus pulex} exposed to propiconazole
-#' during 4 days with constant exposure concentration and 10 days with time variable
+#' during 10 days with time variable
 #' exposure concentration (non-standard pulsed toxicity experiment)
 #'
 #' Survival datasets of laboratory bioassays with \emph{Gammarus pulex} 
 #' freshwater invertebrate exposed to several profiles of concentrations
-#' of one fungicide (propiconazole) (constant & pulsed). 
-#'
+#' of one fungicide (propiconazole). 
 #'
-#' @name propiconazole_varC_Nsurv
+#' @name propiconazole_pulse_exposure
 #' @docType data
-#' @usage data(propiconazole_varC_Nsurv)
-#' @format A data frame with 84 observations on the following four variables:
+#' @usage data(propiconazole_pulse_exposure)
+#' @format A data frame with 74 observations on the following four variables:
```

Comment:
Just to be sure: since there are 74 obs in the joint datasets, and resp. 65 and 44 in the separated datasets, it means that there are some common timepoints between concentration and survival measures, right?